src/components/ComputeModal.js

```diff
@@ -697,7 +698,7 @@ export const ComputeModalBase = ({
         currentRuntime ? Ajax().Runtimes.runtime(currentRuntime.googleProject, currentRuntime.runtimeName).details() : null,
         Ajax()
           .Buckets
-          .getObjectPreview(googleProject, 'terra-docker-image-documentation', 'terra-docker-versions.json', true)
+          .getObjectPreview(googleProject, 'terra-docker-image-documentation', getConfig().terraDockerVersionsFile, true)
```

Comment:
One new question--will this work for paths inside the bucket? If `getConfig().terraDockerVersionsFile` was `myfolder/terra-docker-versions-temp-123.json`, would the getObjectPreview function know to do the small bit of directory traversal to grab the file?